RayTraceAntiXray/src/main/java/com/vanillage/raytraceantixray/util/SchedulerUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.vanillage.raytraceantixray.util;
+
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.plugin.Plugin;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.function.Consumer;
+
+public class SchedulerUtil {
```

Comment:
As already said, it might also be advantageous to wait for `PaperMC/Paper#9360`, then we don't need anything in this class.